ecs/s3.conf

```diff
@@ -0,0 +1,19 @@
+[SERVICE]
+    Grace 30
+
+[INPUT]
+    Name forward
+    unix_path /var/run/fluent.sock
+
+[FILTER]
+    Name record_modifier
+    Match *
+    Record ecs_cluster ${ECS_CLUSTER}
+    Record ecs_task_arn ${ECS_TASK_ARN}
+    Record ecs_task_definition ${ECS_TASK_DEFINITION}
+
+[OUTPUT]
+    Name s3
+    Match *
+    bucket ${BUCKET}
+    region ${S3_REGION}
```

Comment:
please use the settings recommended for scenario with no persistent disk as shown here: https://docs.fluentbit.io/manual/pipeline/outputs/s3#using-s3-without-persisted-disk

also, we need to set S3 key format. I look forward to seeing you idea for that... it should be something that includes the ECS Metadata..

Reply:
so maybe we need support ecs metadata in s3_key_format